tests/test_app_events.py

```diff
@@ -1,66 +1,46 @@
 """Test events occurring on app startup or shutdown."""
 
-import os
 import warnings
 
-import httpx
 import pytest
 
 from app.api import utility as util
+from app.main import settings
 
 
 @pytest.mark.filterwarnings("ignore:.*NB_API_ALLOWED_ORIGINS")
 def test_start_app_without_environment_vars_fails(
-    test_app, monkeypatch, disable_auth
+    test_app, disable_auth, monkeypatch
 ):
     """Given non-existing username and password environment variables, raises an informative RuntimeError."""
-    monkeypatch.delenv(util.GRAPH_USERNAME.name, raising=False)
-    monkeypatch.delenv(util.GRAPH_PASSWORD.name, raising=False)
+    monkeypatch.setattr(settings, "graph_username", None)
+    monkeypatch.setattr(settings, "graph_password", None)
 
     with pytest.raises(RuntimeError) as e_info:
         with test_app:
             pass
     assert (
-        f"could not find the {util.GRAPH_USERNAME.name} and / or {util.GRAPH_PASSWORD.name} environment variables"
+        "could not find the NB_GRAPH_USERNAME and / or NB_GRAPH_PASSWORD environment variables"
         in str(e_info.value)
     )
 
 
-@pytest.mark.filterwarnings("ignore:.*NB_API_ALLOWED_ORIGINS")
-def test_app_with_invalid_environment_vars(
-    test_app, monkeypatch, mock_auth_header, set_mock_verify_token
-):
-    """Given invalid environment variables for the graph, returns a 401 status code."""
-    monkeypatch.setenv(util.GRAPH_USERNAME.name, "something")
-    monkeypatch.setenv(util.GRAPH_PASSWORD.name, "cool")
-
-    def mock_httpx_post(**kwargs):
-        return httpx.Response(status_code=401)
-
-    monkeypatch.setattr(httpx, "post", mock_httpx_post)
-    response = test_app.get("/query", headers=mock_auth_header)
-    assert response.status_code == 401
```

Comment:
I'm a little puzzled what this test was handling before? I think it would be good to have a test for what happens with bad env variables, but I guess that would have to be a real integration test (i.e. with a real un-mocked graph in the back)